src/CUDA.jl

```diff
@@ -64,7 +64,8 @@ include("array.jl")
 
 # compiler libraries
 include("../lib/cupti/CUPTI.jl")
-export CUPTI
+include("../lib/nvperf/NVPERF.jl")
```

Comment:
Why a default module? We only load CUPTI because we use it for `@device_code_sass`. Or do you have similar plans for NVPERF?